js/modules/sunburst.src.js

```diff
@@ -508,14 +508,16 @@ var sunburstOptions = {
         defer: true,
         /**
          * Decides how the data label will be rotated relative to the perimeter
-         * of the sunburst. Valid values are `auto`, `parallel` and
-         * `perpendicular`. When `auto`, the best fit will be computed for the
-         * point.
+         * of the sunburst. Valid values are `auto`, `circular`, `parallel` and
+         * `perpendicular`. When `auto` or `circular`, the best fit will be
+         * computed for the point, where `circular` use the textPath feature.
          *
          * The `series.rotation` option takes precedence over `rotationMode`.
          *
          * @type       {string}
-         * @validvalue ["auto", "perpendicular", "parallel"]
+         * @sample {highcharts} highcharts/plotoptions/sunburst-datalabels-rotationmode-circular/
+         *         rotationMode: circular
```

Comment:
Use natural English for the sample link. Instead of `rotationMode: circular`, write "Circular rotation mode".